pragma/views/endpoints.py

```diff
@@ -24,12 +24,6 @@ def get(self, oaid=None):
             return OpenAnnotation.get(oaid).dict()
         return {"annotations": [oa.dict() for oa in OpenAnnotation.query.all()]}
 
-    @rest
```

Comment:
I removed it because it doesn't actually function as written -- it never calls oa.create(). I agree about making sure the github repo is up to date with the deployed code. :-)

(previously posted from the wrong account)